src/binding/c/misc_api.txt

```diff
@@ -0,0 +1,200 @@
+# vim: set ft=c:
+
+MPI_Aint_add:
+    .desc: Returns the sum of base and disp
+    .skip: ThreadSafe, Fortran, validate-ANY
+    .seealso: MPI_Aint_diff
+    .impl: direct
+/*
+    Notes:
+    MPI_Aint_Add produces a new MPI_Aint value that is equivalent to the sum of the
+    base and disp arguments, where base represents a base address returned by a call
+    to MPI_GET_ADDRESS and disp represents a signed integer displacement. The resulting
+    address is valid only at the process that generated base, and it must correspond
+    to a location in the same object referenced by base. The addition is performed in
+    a manner that results in the correct MPI_Aint representation of the output address,
+    as if the process that originally produced base had called\:
+    .vb
+        MPI_Get_address((char *) base + disp, &result)
+    .ve
+*/
+{
+    MPI_Aint result;
+    MPIR_FUNC_TERSE_STATE_DECL(MPID_STATE_MPI_AINT_ADD);
+
+    MPIR_ERRTEST_INITIALIZED_ORDIE();
+
+    MPIR_FUNC_TERSE_ENTER(MPID_STATE_MPI_AINT_ADD);
+    result = MPID_Aint_add(base, disp);
+    MPIR_FUNC_TERSE_EXIT(MPID_STATE_MPI_AINT_ADD);
+
+    return result;
+}
+
+MPI_Aint_diff:
+    .desc: Returns the difference between addr1 and addr2
+    .skip: ThreadSafe, Fortran, validate-ANY
+    .seealso: MPI_Aint_add
+    .impl: direct
+/*
+    Notes:
+    MPI_Aint_diff produces a new MPI_Aint value that is equivalent to the difference
+    between addr1 and addr2 arguments, where addr1 and addr2 represent addresses
+    returned by calls to MPI_GET_ADDRESS. The resulting address is valid only at the
+    process that generated addr1 and addr2, and addr1 and addr2 must correspond to
+    locations in the same object in the same process. The difference is calculated
+    in a manner that results the signed difference from addr1 to addr2, as if the
+    process that originally produced the addresses had called
+    .vb
+        (char *) addr1 - (char *) addr2
+    .ve
+    on the addresses initially passed to MPI_GET_ADDRESS.
+*/
+{
+    MPI_Aint result;
+    MPIR_FUNC_TERSE_STATE_DECL(MPID_STATE_MPI_AINT_ADD);
+
+    MPIR_ERRTEST_INITIALIZED_ORDIE();
+
+    MPIR_FUNC_TERSE_ENTER(MPID_STATE_MPI_AINT_ADD);
+    result = MPID_Aint_diff(addr1, addr2);
+    MPIR_FUNC_TERSE_EXIT(MPID_STATE_MPI_AINT_ADD);
+
+    return result;
+}
+
+MPI_Get_processor_name:
+    .desc: Gets the name of the processor
+    .skip: ThreadSafe, Fortran
+/*
+    .N ThreadSafe
+
+    .N Fortran
+
+     In Fortran, the character argument should be declared as a character string
+     of 'MPI_MAX_PROCESSOR_NAME' rather than an array of dimension
+     'MPI_MAX_PROCESSOR_NAME'.  That is,
+    .vb
+       character*(MPI_MAX_PROCESSOR_NAME) name
+    .ve
+     rather than
+    .vb
+       character name(MPI_MAX_PROCESSOR_NAME)
+    .ve
+     The two
```

Comment:
It exists in the original code, but can I ask you to remove this "the two"?

Reply:
Removed.